Source/Core/Plugins/EventProcessor/Default/40_RequestInfoPlugin.cs

```diff
@@ -40,36 +40,51 @@ public class RequestInfoPlugin : EventProcessorPluginBase {
                 if (request == null)
                     continue;
 
-                var info = await _parser.ParseAsync(request.UserAgent, context.Project.Id).AnyContext();
-                if (info != null) {
-                    if (!String.Equals(info.UserAgent.Family, "Other")) {
-                        request.Data[RequestInfo.KnownDataKeys.Browser] = info.UserAgent.Family;
-                        if (!String.IsNullOrEmpty(info.UserAgent.Major)) {
-                            request.Data[RequestInfo.KnownDataKeys.BrowserVersion] = String.Join(".", new[] { info.UserAgent.Major, info.UserAgent.Minor, info.UserAgent.Patch }.Where(v => !String.IsNullOrEmpty(v)));
-                            request.Data[RequestInfo.KnownDataKeys.BrowserMajorVersion] = info.UserAgent.Major;
-                        }
-                    }
+                SetClientIPAddress(request, context.EventPostInfo?.IpAddress);
+                await SetBrowserOsAndDeviceFromUserAgent(request, context);
+                
+                context.Event.AddRequestInfo(request.ApplyDataExclusions(exclusions, MAX_VALUE_LENGTH));
+            }
+        }
+
+        private void SetClientIPAddress(RequestInfo request, string clientIPAddress) {
+            if (String.IsNullOrEmpty(clientIPAddress))
+                return;
 
-                    if (!String.Equals(info.Device.Family, "Other"))
-                        request.Data[RequestInfo.KnownDataKeys.Device] = info.Device.Family;
+            if (clientIPAddress.IsLocalHost())
+                return;
```

Comment:
We are returning early if the ip address from the submission request is local host. I'm not 100% sure if want to do this but it would cut down on us having to check for both `::1` and `127.0.0.1`.